exonum/src/blockchain/transaction.rs

```diff
@@ -343,6 +357,54 @@ mod tests {
         static ref EXECUTION_STATUS: Mutex<ExecutionResult> = Mutex::new(Ok(()));
     }
 
+    #[test]
+    fn execution_error_new() {
+        let codes = [0, 1, 100, 255];
+
+        for code in &codes {
```

Comment:
Is there a reason to not use `for code in codes.into_iter()` or `for &code in &codes`?